homeassistant/components/foscam/camera.py

```diff
@@ -107,6 +115,11 @@ def __init__(self, camera: FoscamCamera, config_entry: ConfigEntry) -> None:
         self._rtsp_port = config_entry.data[CONF_RTSP_PORT]
         if self._rtsp_port:
             self._attr_supported_features = CameraEntityFeature.STREAM
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, config_entry.entry_id)},
+            name=config_entry.title,
```

Comment:
`_attr_name` is also set to `config_entry.title`...

Wouldn't it make more sense to enable has_entity_name on this one, and remove both?